apps/dashboard/app/views/batch_connect/sessions/card/_session_time.html.erb

```diff
@@ -0,0 +1,25 @@
+<% time_limit = session.info.wallclock_limit %>
+<% time_used  = session.info.wallclock_time %>
+<p>
+  <% if session.starting? || session.running? %>
+    <% if time_limit && time_used %>
+      <strong> 
+        <%= t('dashboard.batch_connect_sessions_stats_time_remaining') %> 
+      </strong>
+      <%= distance_of_time_in_words(time_limit - time_used, 0, false, :only => [:minutes, :hours], :accumulate_on => :hours) %>
+    <% elsif time_used %>
+      <strong> 
+        <%= t('dashboard.batch_connect_sessions_stats_time_used') %>
+      </strong>
+      <%= distance_of_time_in_words(time_used, 0, false, :only => [:minutes, :hours], :accumulate_on => :hours) %>
+    <% end %>
+  <% else %>
+    <% if time_limit %>
+      <strong> 
+        <%= t('dashboard.batch_connect_sessions_stats_time_used') %>
```

Comment:
This last one here needs to be: `dashboard.batch_connect_sessions_stats_time_requested`  Otherwise while you sit in a queue (i.e., the job hasn't started yet) it says `Time Used: ...` instead of `Time Requested: ...`.